Description: Use rm -f for make clean
 This patch prevents make clean from failing when the source tree is already
 clean.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-16

--- a/wmppp/Makefile
+++ b/wmppp/Makefile
@@ -16,9 +16,9 @@
 
 clean::
 	for i in $(OBJS) ; do \
-		rm $$i; \
+		rm -f $$i; \
 	done
-	rm wmppp getmodemspeed
+	rm -f wmppp getmodemspeed
 
 install::
 	
